scss/base/generic.scss

```diff
@@ -41,3 +41,8 @@ input[type="radio"],
 input[type="checkbox"] {
   outline: 0;
 }
+
+// Hides the default <input type="file"> upload button. A <label> targeting the id of the now hidden input is required.
+input[type="file"] {
+  display: none;
+}
```

Comment:
Could you move `input[type="file"]` into `buttons.scss`?
Because `Input-File` of users who don't want to use `nes-btn` will be hidden.
```html
<label class="nes-btn">
  <span>Upload File</span>
  <input type="file">
</label>
```
```scss
// ./elements/buttons.scss
.nes-btn > input[type="file"] {
  display: none;
}
```

What do you think? 🤔